lib/profile/settings.dart

```diff
@@ -0,0 +1,314 @@
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(Settings());
+}
+
+class Settings extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: SettingsPage(
+        appBarLeading: IconButton(
+          icon: Icon(Icons.arrow_back),
+          onPressed: () {
+            Navigator.pop(context);
+          },
+        ),
+      ),
+    );
+  }
+}
+
+class SettingsPage extends StatelessWidget {
+  final Widget appBarLeading;
+
+  SettingsPage({required this.appBarLeading});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          'Account',
+          style: TextStyle(color: Colors.black),
+        ),
+        backgroundColor: Colors.white,
+        centerTitle: true,
+        leading: appBarLeading,
+      ),
+      body: Container(
+        color: Colors.grey[200],
+        child: Column(
+          children: [
+            SizedBox(height: 20),
+            CircleAvatar(
+              radius: 70,
+              backgroundImage: AssetImage('assets/profile_picture.jpg'),
+            ),
+            SizedBox(height: 20),
+            Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 14.0),
+              child: ClipRRect(
+                borderRadius: BorderRadius.circular(15.0),
+                child: Card(
+                  elevation: 0,
+                  color: Colors.white,
+                  child: Column(
+                    children: [
+                      ListTile(
```

Comment:
All the list tiles on this page should be initialized in a for loop to reduce code repetitiveness and increase readability

I don't know the exact methods/widgets used in flutter for this but I'm sure there is something similar to render all the Sections and Tiles within just ~50 lines

pseudocode:

```//Build sections here based on incoming data from user or have them defined staticly
//I think a nested map would work to hold the sections and field values
Map<string, Map<string, string>> mapSection_FieldValue = new Map<string, Map<string,string>>()
// Example of what map would look like once constructed:
      [
           "Details":  {
                     "Email": "abc@123.com",
                     "Name": "khoas1234567890",
                  },
           "Preferences": {
                     "Theme": "light",
                     "PictureBorder": "square",
                     etc
           }
      ]

//Then would loop through the nested map
for (section in mapSection_FieldValue.keySet()) {
     //render section name based on section key
     Text(section.key())
     //loop through nested map
     for (field in section.value()) {
        //Initialize list tile for each value within the section
        ListTile(title: field.key(), value: field.value(), trailing: Icon(Icons.arrow_forward))
     }
}